### ISSUE TITLE
What can I do to help?

### ISSUE THREAD

**Issue by cosmotek** [carloslfu/figma-app-linux#20]
Hey! Love that you built this, what can I do to help? If you are still in need of something for automated builds I would love to help, I might be able to provide some hardware, or setup something CI to help get the job done.

I'm an Arch user, so I would love to see more of these up-to-date AppImages

**Comment by carloslfu** [comment]
Hi @cosmotek, yeah I would love to have automated builds on this project. I really appreciate your initiative, thanks!

**Comment by cosmotek** [comment]
Definitely! I am working on a little Concourse CI setup on DO for some other projects, so I will just add a pipeline for this project as well. I'll see if I can't setup Github integration that would allow you to have some control over the pipeline.

**Comment by carloslfu** [comment]
Great! Thanks, @cosmotek.
